### ISSUE TITLE
Release of models for COCO object detection and segmentation

### ISSUE THREAD

**Issue by hector-gr** [facebookresearch/hiera#18]
Thank you for such great work!

Will you also released trained models for COCO object detection and segmentation, and inference/training scripts?

**Comment by dbolya** [comment]
COCO detection / segmentation scripts won't be released in this repo, though it might be released in detectron2 much later. I'll update this issue if it does.

**Comment by VGrondin** [comment]
Hello, I have a similar question as I would be interested in doing research with the finetuned model on INaturalist data. Any plan on releasing the model weights?

**Comment by wenhwu** [comment]
@dbolya ，Hello, I have implemented a basic version of HieraDet for object detection in Detectron2. Thanks to the advantages of Hiera's design, the training speed is significantly faster than ViTDet. However, the accuracy of my implementation is quite low. Do you have any plans to release the official detection and segmentation code of HieraDet in Detectron2? Look forward to your code, thanks
